docs/history.rst

```diff
@@ -36,8 +36,9 @@ injecting said dependencies at runtime. We have to credit the great work by @wic
 The primary differences between PEX and shiv are:
 
 * ``shiv`` completey avoids the use of ``pkg_resources``. If it is included by a transitive
-  dependency, the performance implications are mitigated by limiting the length of ``sys.path`` and
-  always including the `-s <https://docs.python.org/3/using/cmdline.html#cmdoption-s>`_ and
+  dependency, the performance implications are mitigated by limiting the length of ``sys.path``.
+  Internally, we always including the
+  `-s <https://docs.python.org/3/using/cmdline.html#cmdoption-s>`_ and
```

Comment:
Is this still true then?

Reply:
Notice that I added "internally we..." 😉